examples/KeyWallet/MintsAndSendsToken.js

```diff
@@ -0,0 +1 @@
+exports.alwaysMints = require("Scripts/always-mints.plutus");
```

Comment:
This won't work in a nodejs environment, right? I think you need to use the `fs.readSync` trick in the other examples. Otherwise you could import `Examples.AlwaysMints.alwaysMintsPolicy`